docs/tasks.md

```diff
@@ -6,6 +6,8 @@ The `TaskList` class implements methods to build and execute a set of tasks with
 ### AddTask
 `AddTask` is a templated variadic function that takes the task function to be executed, the task dependencies (see `TaskID` below), and the arguments to the task function as it's arguments.  All arguments are captured by value in a lambda for later execution.
 
+When adding functions that are non-static class member functions, a slightly different interface is required.  The first argument should be the class-name-scoped name of the function.  For example, for a function named `DoSomething` in class `SomeClass`, the first argument would be `&SomeClass::DoSomething`.  The second argument should be a pointer to the object that should invoke this member function.  Finally, the dependencies and function arguments should be provided as described above.
+
```

Comment:
Some example calls might be useful.

Reply:
advection example now uses this with refactored Container tasks.  doc points to the example.  good?

Reply:
Yep. Thanks for adding that!